gitignore

```diff
@@ -9,3 +9,4 @@ tags
 !tags/
 tmp/**/*
 !tmp/cache/.keep
+local_bin/
```

Comment:
I could go either way. I checked it in on the docker project I was on, but it made me uneasy - maybe because the name is "*local*_bin".